attributes/default.rb

```diff
@@ -19,7 +19,7 @@
 
 return unless %w(darwin windows linux).include?(node['os'])
 
-default['vagrant']['version']     = '1.9.7'
+default['vagrant']['version']     = '2.0.3'
```

Comment:
2.0.3 was planned for the next version. Cool.  Folks told me 1.9.7 had issues and was likely the cause of most of the install problems.